docs/_data/navigation.yml

```diff
@@ -0,0 +1,7 @@
+main:
+  - title: "Quick-Start Guide"
+    url: https://github.com/scale-it/algo-builder#installation
+  - title: "API Docs"
+    url: https://scale-it.github.io/algo-builder/ # TODO: Update to custom api readme link
+  - title: "Index"
+    url: /guide/README
```

Comment:
Please add links to:
-  _Home_ (https://scale-it.github.io/algo-builder/)
- _Dapp Templates_ repository


The order of the links should be: Home, Quick Start, Guide, API,

Reply:
Done.  BDW - The title in top bar (`Algo Builder` on LHS) also points to HOME (`/`)